test/e2e/performanceprofile/functests/utils/cgroup/v1/v1.go

```diff
@@ -0,0 +1,38 @@
+package v1
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/client-go/kubernetes"
+
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	"github.com/openshift/cluster-node-tuning-operator/test/e2e/performanceprofile/functests/utils/cgroup/config"
+	"github.com/openshift/cluster-node-tuning-operator/test/e2e/performanceprofile/functests/utils/pods"
+)
+
+type Manager struct {
+}
+
+func (m *Manager) GetConfig(ctx context.Context, c *kubernetes.Clientset, pod *corev1.Pod, containerName string) (*config.Config, error) {
+	var cmd []string
+	cfg := &config.Config{}
+	cmd = []string{
+		"/bin/cat",
+		config.CgroupMountPoint + "/cpuset/cpuset.cpus",
+		config.CgroupMountPoint + "/cpu/cpu.cfs_quota_us",
+		config.CgroupMountPoint + "/cpu/cpu.cfs_period_us",
+	}
+	b, err := pods.ExecCommandOnPod(c, pod, containerName, cmd)
+	output := strings.Split(string(b), "\r\n")
+	cfg.CPUSet = output[0]
+	cfg.Quota = output[1]
+	cfg.Period = output[2]
+	if err != nil {
```

Comment:
you may want to do this earlier, because if `Split` doesn't yield 3 values, we will hit an error. OTOH `Split` likely fails only iff `err != nil`.

Reply:
Thanks. fixed.